lib/tasks/webpacker/compile.rake

```diff
@@ -1,35 +1,24 @@
+require "webpacker/configuration"
 REGEX_MAP = /\A.*\.map\z/
 
 namespace :webpacker do
   desc "Compile javascript packs using webpack for production with digests"
-  task :compile => :environment do
-    dist_dir = Rails.application.config.x.webpacker[:packs_dist_dir]
-    result   = `WEBPACK_DIST_DIR=#{dist_dir} NODE_ENV=production ./bin/webpack --json`
+  task compile: ["webpacker:install:verify", :environment] do
+    result = `NODE_ENV=production ./bin/webpack`
```

Comment:
It serves the purpose without `--json` option provided, which is showing the errors. Perhaps, we just get rid of the error parsing underneath.